packages/pyright-internal/src/tests/common.test.ts

```diff
@@ -6,7 +6,6 @@
 
 import * as assert from 'assert';
 
-import { computeCompletionSimilarity } from '../common/stringUtils';
```

Comment:
can you leave this test in since this still exist in pyright?

Reply:
oops. ignore, you have the test in other places.

Reply:
I moved it to `tests/stringUtils.test.ts`, since that file already existed.